mp/src/game/server/neo/neo_player.cpp

```diff
@@ -1056,7 +1059,75 @@ void CNEO_Player::PlayerDeathThink()
 	{
 		m_bEnterObserver = true;
 	}
-	BaseClass::PlayerDeathThink();
+
+	PlayerDeathThinkInner();
```

Comment:
The observer free fly mode feels jittery. This seems to be because we're missing the if `!IsObserver()` guard, as per: 

https://github.com/NeotokyoRebuild/neo/blob/6b31ee46d966c8a4c831ad282f794ba5fb99cf51/mp/src/game/server/hl2mp/hl2mp_player.cpp#L606-L612

To reproduce:

* Join a local map (e.g. `map nt_oilstain`)
* Join the spectator team
* Hit the space bar a few times to enter the free fly mode, so you can move around
* Fly around the map, and focus your eyes on some target while doing strafes/turns around it

What should happen:
* The game feels smooth and "high fps"

What actually happens:
* The game feels laggy/jittery as you're flying around the target point, like it's skipping frames